x-pack/plugins/cases/public/components/all_cases/all_cases_list.test.tsx

```diff
@@ -504,7 +501,7 @@ describe('AllCasesListGeneric', () => {
       expect(useGetCasesMock).toHaveBeenLastCalledWith({
         filterOptions: {
           ...DEFAULT_FILTER_OPTIONS,
-          searchFields: [],
+          searchFields: ['title', 'description'],
```

Comment:
there was an issue, mocking containers/use_get_cases would also clear DEFAULT_QUERY_PARAMS. I moved DEFAULT_QUERY_PARAMS to the constants file in order to avoid this issue and updated the expected objects

Thanks Christos!